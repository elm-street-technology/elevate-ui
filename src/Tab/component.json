{
  "description": "A component meant to be wrapped in `<TabNavigation />` that allows you to create a \"tab-style\" navigation.",
  "displayName": "Tab",
  "methods": [],
  "props": {
    "classes": {
      "flowType": {
        "name": "Object"
      },
      "required": true,
      "description": ""
    },
    "element": {
      "flowType": {
        "name": "any"
      },
      "required": true,
      "description": "Accepts any element to be used as the component base."
    },
    "className": {
      "flowType": {
        "name": "string"
      },
      "required": false,
      "description": "Accepts any classNames to be passed down to the component."
    },
    "children": {
      "flowType": {
        "name": "any"
      },
      "required": true,
      "description": ""
    },
    "isActive": {
      "flowType": {
        "name": "boolean"
      },
      "required": true,
      "description": "Should the tab show an active state."
    }
  }
}