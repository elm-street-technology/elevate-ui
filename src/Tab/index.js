// @flow
import React from "react";
import withStyles from "../withStyles";
import classNames from "classnames";

type $Props = {
  classes: Object,
  /**
   * Accepts any element to be used as the component base.
   */
  element: any,
  /**
   * Accepts any classNames to be passed down to the component.
   */
  className?: string,
  children: any,
  /**
   * Should the tab show an active state.
   */
  isActive: boolean,
};

/**
 * A component meant to be wrapped in `<TabNavigation />` that allows you to create a "tab-style" navigation.
 */
const Tab = ({
  classes,
  element,
  children,
  className,
  isActive,
  ...rest
}: $Props) => {
  const Element = element || "button";
  return (
    <Element
      {...rest}
      className={classNames({
        [classes.root]: true,
        [classes.active]: isActive,
        className,
      })}
    >
      {children}
    </Element>
  );
};

const styles = (theme) => ({
  root: {
    padding: "24px",
    fontSize: "15px",
    fontWeight: "600",
    letterSpacing: ".5px",
    lineHeight: 1,
    textTransform: "uppercase",
    borderLeft: `4px solid ${theme.colors.gray["100"]}`,
    transition: theme.transitions.default,
    color: theme.colors.gray["600"],
    textDecoration: "none",
    "&:hover": {
      color: theme.colors.black,
    },
    [theme.breakpoints(670)]: {
      fontSize: "14px",
      padding: "12px 24px",
      borderBottom: `2px solid ${theme.colors.gray["100"]}`,
      borderLeft: "none",
    },
  },
  active: {
    color: theme.colors.black,
    borderLeft: `4px solid ${theme.colors.tertiary["300"]}`,
    [theme.breakpoints(670)]: {
      borderBottom: `2px solid ${theme.colors.tertiary["300"]}`,
      borderLeft: "none",
    },
  },
});

export default withStyles(styles)(Tab);
