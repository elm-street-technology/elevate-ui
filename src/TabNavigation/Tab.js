// @flow
import React from "react";
import withStyles from "../withStyles";
import classNames from "classnames";

type $Props = {
  classes: Object,
  Element: any,
  className?: string,
  children: any,
  isActive: boolean,
};

const Tab = ({ classes, Element, children, isActive, ...rest }: $Props) => (
  <Element
    {...rest}
    className={classNames({ [classes.root]: true, [classes.active]: isActive })}
  >
    {children}
  </Element>
);

const styles = (theme) => ({
  root: {
    padding: "24px",
    fontSize: "15px",
    fontWeight: "600",
    letterSpacing: ".5px",
    textTransform: "uppercase",
    borderLeft: `4px solid ${theme.colors.gray["100"]}`,
    transition: theme.transitions.default,
    color: theme.colors.gray["600"],
    textDecoration: "none",
    "&:hover": {
      color: theme.colors.black,
    },
    [theme.breakpoints(670)]: {
      fontSize: "14px",
      padding: "12px 24px",
      borderBottom: `2px solid ${theme.colors.gray["100"]}`,
      borderLeft: "none",
    },
  },
  active: {
    color: theme.colors.black,
    borderLeft: `4px solid ${theme.colors.tertiary["300"]}`,
    [theme.breakpoints(670)]: {
      borderBottom: `2px solid ${theme.colors.tertiary["300"]}`,
      borderLeft: "none",
    },
  },
});

export default withStyles(styles)(Tab);
