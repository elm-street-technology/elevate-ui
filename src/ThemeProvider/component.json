{
  "description": "",
  "displayName": "ThemeProvider",
  "methods": [],
  "props": {
    "withReset": {
      "defaultValue": {
        "value": "true",
        "computed": false
      },
      "required": false
    }
  }
}