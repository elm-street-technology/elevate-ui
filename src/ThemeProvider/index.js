// @flow
import React from "react";
import injectSheet, { ThemeProvider as JSSThemeProvider } from "react-jss";
import merge from "lodash/merge";

const colors = {
  primary: "#f15953",
  primaryDark: "#d24f45",
  primaryGlow: "#d24f45", // todo
  secondary: "#55c3ba",
  secondaryDark: "#44b0a7",
  tertiary: "#2e7fc2",
  quaternary: "#ffc629",
  white: "#fff",
  gray050: "#fbfcfd",
  gray100: "#edf1f4",
  gray200: "#dde2e7",
  gray300: "#ccd2d8",
  gray400: "#b3bac1",
  gray500: "#888f96",
  gray600: "#60686f",
  gray700: "#4b565f",
  gray800: "#5a5b5c",
  gray900: "#232c35",
  black: "#11181e",
  error: "#9c2929",
  facebook: "#3b5998",
  google: "#db4437",
  linkedin: "#007bb5",
  pinterest: "#bd081c",
  twitter: "#1da1f2",
};

const alertColors = {
  error: {
    color: "#9c2929",
    backgroundColor: "#fcf2f2",
    borderColor: "#f6dddd",
  },
  info: {
    color: "#0c5460",
    backgroundColor: "#d1ecf1",
    borderColor: "#bee5eb",
  },
  success: {
    color: "#246761",
    backgroundColor: "#ecf8f7",
    borderColor: "#d2e8e6",
  },
  warning: {
    color: "#856404",
    backgroundColor: "#fff3cd",
    borderColor: "#ffeeba",
  },
};

const typography = {
  bodyColor: "#2E2E35",
  fontFamily:
    "'Open Sans', -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif",
  fontSize: "1rem",
};

const transitions = {
  default: "all 200ms linear",
};

const breakpoints = (minWidth: number) => {
  if (!minWidth) {
    return new Error("Must enter a valid minWidth to use theme breakpoints");
  }
  return `@media (min-width: ${minWidth}px)`;
};

const globalBorderRadius = "4px";
const globalBoxShadow =
  "0 2px 5px 0 rgba(0,0,0,0.16), 0 2px 10px 0 rgba(0,0,0,0.12)";
const globalPadding = {
  padding: "12px",
  [breakpoints.tabletPortrait]: {
    padding: "16px",
  },
  [breakpoints.tabletLandscape]: {
    padding: "20px",
  },
  [breakpoints.desktop]: {
    padding: "24px",
  },
};

const shadowKeyUmbraOpacity = 0.2;
const shadowKeyPenumbraOpacity = 0.14;
const shadowAmbientShadowOpacity = 0.12;

function createShadow(...px) {
  return [
    `${px[0]}px ${px[1]}px ${px[2]}px ${
      px[3]
    }px rgba(0, 0, 0, ${shadowKeyUmbraOpacity})`,
    `${px[4]}px ${px[5]}px ${px[6]}px ${
      px[7]
    }px rgba(0, 0, 0, ${shadowKeyPenumbraOpacity})`,
    `${px[8]}px ${px[9]}px ${px[10]}px ${
      px[11]
    }px rgba(0, 0, 0, ${shadowAmbientShadowOpacity})`,
  ].join(",");
}

const shadows = [
  createShadow(0, 1, 3, 0, 0, 1, 1, 0, 0, 2, 1, -1),
  createShadow(0, 1, 5, 0, 0, 2, 2, 0, 0, 3, 1, -2),
  createShadow(0, 1, 8, 0, 0, 3, 4, 0, 0, 3, 3, -2),
  createShadow(0, 2, 4, -1, 0, 4, 5, 0, 0, 1, 10, 0),
  createShadow(0, 3, 5, -1, 0, 5, 8, 0, 0, 1, 14, 0),
  createShadow(0, 3, 5, -1, 0, 6, 10, 0, 0, 1, 18, 0),
  createShadow(0, 4, 5, -2, 0, 7, 10, 1, 0, 2, 16, 1),
  createShadow(0, 5, 5, -3, 0, 8, 10, 1, 0, 3, 14, 2),
  createShadow(0, 5, 6, -3, 0, 9, 12, 1, 0, 3, 16, 2),
  createShadow(0, 6, 6, -3, 0, 10, 14, 1, 0, 4, 18, 3),
  createShadow(0, 6, 7, -4, 0, 11, 15, 1, 0, 4, 20, 3),
  createShadow(0, 7, 8, -4, 0, 12, 17, 2, 0, 5, 22, 4),
  createShadow(0, 7, 8, -4, 0, 13, 19, 2, 0, 5, 24, 4),
  createShadow(0, 7, 9, -4, 0, 14, 21, 2, 0, 5, 26, 4),
  createShadow(0, 8, 9, -5, 0, 15, 22, 2, 0, 6, 28, 5),
  createShadow(0, 8, 10, -5, 0, 16, 24, 2, 0, 6, 30, 5),
  createShadow(0, 8, 11, -5, 0, 17, 26, 2, 0, 6, 32, 5),
  createShadow(0, 9, 11, -5, 0, 18, 28, 2, 0, 7, 34, 6),
  createShadow(0, 9, 12, -6, 0, 19, 29, 2, 0, 7, 36, 6),
  createShadow(0, 10, 13, -6, 0, 20, 31, 3, 0, 8, 38, 7),
  createShadow(0, 10, 13, -6, 0, 21, 33, 3, 0, 8, 40, 7),
  createShadow(0, 10, 14, -6, 0, 22, 35, 3, 0, 8, 42, 7),
  createShadow(0, 11, 14, -7, 0, 23, 36, 3, 0, 9, 44, 8),
  createShadow(0, 11, 15, -7, 0, 24, 38, 3, 0, 9, 46, 8),
];

const zIndex = {
  buttonBase: 0,
  buttonChildren: 1,
  selectDropdown: 2,
  tableLoading: -1,
  tableLoadingActive: 2,
  tableResizer: 10,
  tooltip: 1000,
  modal: 2000,
};

const defaultTheme = {
  alertColors,
  breakpoints,
  colors,
  globalBorderRadius,
  globalBoxShadow,
  globalPadding,
  shadows,
  transitions,
  typography,
  zIndex,
};

const GlobalsAndReset = injectSheet((theme) => ({
  "@global": {
    /* Eric Meyer Reset v2.0 */
    /*Â https://meyerweb.com/eric/tools/css/reset/ */
    "html, body, div, span, applet, object, iframe, h1, h2, h3, h4, h5, h6, p, blockquote, pre, a, abbr, acronym, address, big, cite, code, del, dfn, em, img, ins, kbd, q, s, samp, small, strike, strong, sub, sup, tt, var, b, u, i, center, dl, dt, dd, ol, ul, li, fieldset, form, label, legend, table, caption, tbody, tfoot, thead, tr, th, td, article, aside, canvas, details, embed, figure, figcaption, footer, header, hgroup, menu, nav, output, ruby, section, summary, time, mark, audio, video": {
      margin: "0",
      padding: "0",
      border: "0",
      fontSize: "100%",
      font: "inherit",
      verticalAlign: "baseline",
    },
    "article, aside, details, figcaption, figure, footer, header, hgroup, menu, nav, section": {
      display: "block",
    },
    body: {
      lineHeight: "1",
    },
    "ol, ul": {
      listStyle: "none",
    },
    "blockquote, q": {
      quotes: "none",
    },
    "blockquote:before, blockquote:after, q:before, q:after": {
      content: "''",
      content: "none", // eslint-disable-line
    },
    table: {
      borderCollapse: "collapse",
      borderSpacing: "0",
    },
    /* Globals */
    "html, body": {
      width: "100%",
      minHeight: "100%",
      fontFamily: theme.typography.fontFamily || "sans-serif",
      fontSize: theme.typography.fontSize || "1rem",
      fontWeight: "500",
      color: theme.typography.bodyColor || "#000",
      backgroundColor: "#fafafa",
      "-webkit-font-smoothing": "antialiased",
      "-moz-osx-font-smoothing": "grayscale",
      margin: "0",
      padding: "0",
    },
    /* Box-sizing border-box */
    /* https://www.paulirish.com/2012/box-sizing-border-box-ftw/ */
    html: {
      boxSizing: "border-box",
      background: "transparent",
    },
    "*, *:before, *:after": {
      boxSizing: "inherit",
    },
    /* Reset `button` to nothing */
    /* https://gist.github.com/chrisheninger/a860f87ef4e529b2df606768b97665a8 */
    button: {
      display: "inline-block",
      "-webkit-appearance": "none",
      "-moz-appearance": "none",
      userSelect: "none",
      textDecoration: "none",
      fontFamily: "inherit",
      fontSize: "inherit",
      color: "inherit",
      cursor: "pointer",
      border: "none",
      backgroundColor: "transparent",
      borderColor: "transparent",
      padding: "0",
      margin: "0",
      "&:disabled": {
        opacity: 0.6,
        cursor: "not-allowed",
      },
    },
  },
}))(({ children }) => children);

const ThemeProvider = ({ children, theme, withReset = true, ...rest }: any) => {
  const mergedTheme = theme ? merge(defaultTheme, theme) : defaultTheme;
  return (
    <JSSThemeProvider theme={mergedTheme} {...rest}>
      {withReset ? <GlobalsAndReset>{children}</GlobalsAndReset> : children}
    </JSSThemeProvider>
  );
};

export default ThemeProvider;
