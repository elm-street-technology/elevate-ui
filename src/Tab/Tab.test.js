import React from "react";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";
import sinon from "sinon";

import ThemeProvider from "../ThemeProvider";
import Tab from "./";

const WrappedTab = (props) => (
  <ThemeProvider>
    <Tab {...props}>{props.children}</Tab>
  </ThemeProvider>
);

test("renders without crashing, matches the snapshot", () => {
  const component = renderer.create(
    <WrappedTab isActive={true} element="a" href="https://elevate-ui.com">
      Elevate UI
    </WrappedTab>
  );
  const componentJSON = component.toJSON();
  expect(componentJSON).toMatchSnapshot();
});

test("fires onClick function", () => {
  const onButtonClick = sinon.spy();
  const component = shallow(
    <WrappedTab
      id="test-tab"
      onClick={onButtonClick}
      element="button"
      type="button"
    >
      Save
    </WrappedTab>
  );
  component.find("#test-tab").simulate("click");
  expect(onButtonClick.calledOnce).toEqual(true);
});
