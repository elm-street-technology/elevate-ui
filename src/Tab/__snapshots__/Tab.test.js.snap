// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing, matches the snapshot 1`] = `
<a
  className="Tab-root-0-1-2 Tab-active-0-1-3"
  href="https://elevate-ui.com"
  theme={
    Object {
      "alertColors": Object {
        "error": Object {
          "backgroundColor": "#ffeded",
          "borderColor": "#FFD4D4",
          "color": "#610404",
        },
        "info": Object {
          "backgroundColor": "#E9F5FF",
          "borderColor": "#D6E4F0",
          "color": "#023052",
        },
        "success": Object {
          "backgroundColor": "#f4fff5",
          "borderColor": "#D4F4D4",
          "color": "#043E06",
        },
        "warning": Object {
          "backgroundColor": "#FFFCF0",
          "borderColor": "#FFF2CE",
          "color": "#672F0D",
        },
      },
      "borders": Object {
        "default": "1px solid #dde2e7",
      },
      "breakpoints": [Function],
      "colors": Object {
        "black": "#11181e",
        "elevateRed": Object {
          "050": "#ffeded",
          "100": "#FFE4E4",
          "200": "#FFD4D4",
          "300": "#F99E9B",
          "400": "#F17772",
          "500": "#F15953",
          "600": "#D2413B",
          "700": "#A82B26",
          "800": "#821410",
          "900": "#610404",
        },
        "error": "#9c2929",
        "facebook": "#3b5998",
        "google": "#db4437",
        "gray": Object {
          "050": "#fbfcfd",
          "100": "#edf1f4",
          "200": "#dde2e7",
          "300": "#ccd2d8",
          "400": "#b3bac1",
          "500": "#888f96",
          "600": "#60686f",
          "700": "#4b565f",
          "800": "#3E464F",
          "900": "#232c35",
        },
        "gray050": "#fbfcfd",
        "gray100": "#edf1f4",
        "gray200": "#dde2e7",
        "gray300": "#ccd2d8",
        "gray400": "#b3bac1",
        "gray500": "#888f96",
        "gray600": "#60686f",
        "gray700": "#4b565f",
        "gray800": "#3E464F",
        "gray900": "#232c35",
        "green": Object {
          "050": "#f4fff5",
          "100": "#E0F9E0",
          "200": "#D4F4D4",
          "300": "#9EE9A1",
          "400": "#6DCA71",
          "500": "#4CAF50",
          "600": "#37983A",
          "700": "#1E7C21",
          "800": "#105E12",
          "900": "#043E06",
        },
        "linkedin": "#007bb5",
        "orange": Object {
          "050": "#FEF5F0",
          "100": "#FFF0E7",
          "200": "#FFEADE",
          "300": "#FFC4A3",
          "400": "#F5A375",
          "500": "#EF8E58",
          "600": "#C65D21",
          "700": "#8C3D10",
          "800": "#77340D",
          "900": "#572508",
        },
        "pink": Object {
          "050": "#FFF2F9",
          "100": "#FDE5F2",
          "200": "#F9DBEC",
          "300": "#FF94C2",
          "400": "#F25087",
          "500": "#E91E63",
          "600": "#C91854",
          "700": "#A91354",
          "800": "#800C49",
          "900": "#5C0B33",
        },
        "pinterest": "#bd081c",
        "primary": Object {
          "050": "#ffeded",
          "100": "#FFE4E4",
          "200": "#FFD4D4",
          "300": "#F99E9B",
          "400": "#F17772",
          "500": "#F15953",
          "600": "#D2413B",
          "700": "#A82B26",
          "800": "#821410",
          "900": "#610404",
        },
        "primaryDark": "#610404",
        "primaryLight": "#ffeded",
        "purple": Object {
          "050": "#F7F3FF",
          "100": "#F1EAFF",
          "200": "#EAE2F8",
          "300": "#A081D9",
          "400": "#835BCD",
          "500": "#673AB7",
          "600": "#51279B",
          "700": "#421987",
          "800": "#34126F",
          "900": "#240754",
        },
        "quaternary": "#F7C948",
        "quaternaryDark": "#672F0D",
        "quaternaryLight": "#FFFCF0",
        "secondary": Object {
          "050": "#F3FFFE",
          "100": "#EBFAF9",
          "200": "#DAF2F1",
          "300": "#9EE5E0",
          "400": "#68D0C8",
          "500": "#55C3BA",
          "600": "#2EA59B",
          "700": "#168C82",
          "800": "#086B62",
          "900": "#014A44",
        },
        "secondaryDark": "#014A44",
        "secondaryLight": "#F3FFFE",
        "tertiary": Object {
          "050": "#E9F5FF",
          "100": "#E1EDF8",
          "200": "#D6E4F0",
          "300": "#80B7E4",
          "400": "#4891CD",
          "500": "#2E7FC2",
          "600": "#19629E",
          "700": "#105188",
          "800": "#073E6C",
          "900": "#023052",
        },
        "tertiaryDark": "#023052",
        "tertiaryLight": "#E9F5FF",
        "twitter": "#1da1f2",
        "white": "#fff",
        "yellow": Object {
          "050": "#FFFCF0",
          "100": "#FFF8E5",
          "200": "#FFF2CE",
          "300": "#FFE292",
          "400": "#F7CF61",
          "500": "#F7C948",
          "600": "#C99400",
          "700": "#905710",
          "800": "#844118",
          "900": "#672F0D",
        },
      },
      "globalBorderRadius": "4px",
      "globalBoxShadow": "0 2px 5px 0 rgba(0,0,0,0.16), 0 2px 10px 0 rgba(0,0,0,0.12)",
      "globalPadding": Object {
        "padding": "12px",
        "undefined": Object {
          "padding": "24px",
        },
      },
      "shadows": Array [
        "0px 1px 3px 0px rgba(0, 0, 0, 0.2),0px 1px 1px 0px rgba(0, 0, 0, 0.14),0px 2px 1px -1px rgba(0, 0, 0, 0.12)",
        "0px 1px 5px 0px rgba(0, 0, 0, 0.2),0px 2px 2px 0px rgba(0, 0, 0, 0.14),0px 3px 1px -2px rgba(0, 0, 0, 0.12)",
        "0px 1px 8px 0px rgba(0, 0, 0, 0.2),0px 3px 4px 0px rgba(0, 0, 0, 0.14),0px 3px 3px -2px rgba(0, 0, 0, 0.12)",
        "0px 2px 4px -1px rgba(0, 0, 0, 0.2),0px 4px 5px 0px rgba(0, 0, 0, 0.14),0px 1px 10px 0px rgba(0, 0, 0, 0.12)",
        "0px 3px 5px -1px rgba(0, 0, 0, 0.2),0px 5px 8px 0px rgba(0, 0, 0, 0.14),0px 1px 14px 0px rgba(0, 0, 0, 0.12)",
        "0px 3px 5px -1px rgba(0, 0, 0, 0.2),0px 6px 10px 0px rgba(0, 0, 0, 0.14),0px 1px 18px 0px rgba(0, 0, 0, 0.12)",
        "0px 4px 5px -2px rgba(0, 0, 0, 0.2),0px 7px 10px 1px rgba(0, 0, 0, 0.14),0px 2px 16px 1px rgba(0, 0, 0, 0.12)",
        "0px 5px 5px -3px rgba(0, 0, 0, 0.2),0px 8px 10px 1px rgba(0, 0, 0, 0.14),0px 3px 14px 2px rgba(0, 0, 0, 0.12)",
        "0px 5px 6px -3px rgba(0, 0, 0, 0.2),0px 9px 12px 1px rgba(0, 0, 0, 0.14),0px 3px 16px 2px rgba(0, 0, 0, 0.12)",
        "0px 6px 6px -3px rgba(0, 0, 0, 0.2),0px 10px 14px 1px rgba(0, 0, 0, 0.14),0px 4px 18px 3px rgba(0, 0, 0, 0.12)",
        "0px 6px 7px -4px rgba(0, 0, 0, 0.2),0px 11px 15px 1px rgba(0, 0, 0, 0.14),0px 4px 20px 3px rgba(0, 0, 0, 0.12)",
        "0px 7px 8px -4px rgba(0, 0, 0, 0.2),0px 12px 17px 2px rgba(0, 0, 0, 0.14),0px 5px 22px 4px rgba(0, 0, 0, 0.12)",
        "0px 7px 8px -4px rgba(0, 0, 0, 0.2),0px 13px 19px 2px rgba(0, 0, 0, 0.14),0px 5px 24px 4px rgba(0, 0, 0, 0.12)",
        "0px 7px 9px -4px rgba(0, 0, 0, 0.2),0px 14px 21px 2px rgba(0, 0, 0, 0.14),0px 5px 26px 4px rgba(0, 0, 0, 0.12)",
        "0px 8px 9px -5px rgba(0, 0, 0, 0.2),0px 15px 22px 2px rgba(0, 0, 0, 0.14),0px 6px 28px 5px rgba(0, 0, 0, 0.12)",
        "0px 8px 10px -5px rgba(0, 0, 0, 0.2),0px 16px 24px 2px rgba(0, 0, 0, 0.14),0px 6px 30px 5px rgba(0, 0, 0, 0.12)",
        "0px 8px 11px -5px rgba(0, 0, 0, 0.2),0px 17px 26px 2px rgba(0, 0, 0, 0.14),0px 6px 32px 5px rgba(0, 0, 0, 0.12)",
        "0px 9px 11px -5px rgba(0, 0, 0, 0.2),0px 18px 28px 2px rgba(0, 0, 0, 0.14),0px 7px 34px 6px rgba(0, 0, 0, 0.12)",
        "0px 9px 12px -6px rgba(0, 0, 0, 0.2),0px 19px 29px 2px rgba(0, 0, 0, 0.14),0px 7px 36px 6px rgba(0, 0, 0, 0.12)",
        "0px 10px 13px -6px rgba(0, 0, 0, 0.2),0px 20px 31px 3px rgba(0, 0, 0, 0.14),0px 8px 38px 7px rgba(0, 0, 0, 0.12)",
        "0px 10px 13px -6px rgba(0, 0, 0, 0.2),0px 21px 33px 3px rgba(0, 0, 0, 0.14),0px 8px 40px 7px rgba(0, 0, 0, 0.12)",
        "0px 10px 14px -6px rgba(0, 0, 0, 0.2),0px 22px 35px 3px rgba(0, 0, 0, 0.14),0px 8px 42px 7px rgba(0, 0, 0, 0.12)",
        "0px 11px 14px -7px rgba(0, 0, 0, 0.2),0px 23px 36px 3px rgba(0, 0, 0, 0.14),0px 9px 44px 8px rgba(0, 0, 0, 0.12)",
        "0px 11px 15px -7px rgba(0, 0, 0, 0.2),0px 24px 38px 3px rgba(0, 0, 0, 0.14),0px 9px 46px 8px rgba(0, 0, 0, 0.12)",
      ],
      "spacing": Object {
        "unit": 16,
      },
      "transitions": Object {
        "accordion": "all 2s ease-in-out",
        "default": "all 200ms linear",
      },
      "typography": Object {
        "bodyColor": "#2E2E35",
        "fontFamily": "'Open Sans', -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif",
        "fontSize": "1rem",
      },
      "zIndex": Object {
        "buttonBase": 0,
        "buttonChildren": 1,
        "dropdown": 1500,
        "modal": 2000,
        "selectDropdown": 2,
        "tableLoading": -1,
        "tableLoadingActive": 2,
        "tableResizer": 10,
        "tooltip": 1000,
      },
    }
  }
>
  Elevate UI
</a>
`;
